training.rb

```diff
@@ -0,0 +1,5 @@
+module Training
```

Comment:
This method is just in the ApprenticeTeacher class. Why make it a module? It is not a shared action across multiple classes.